tutorial-binder-with-templates/package.json

```diff
@@ -6,38 +6,38 @@
       "@vaadin/router": "^1.6.0",
       "@polymer/polymer": "3.2.0",
       "@webcomponents/webcomponentsjs": "^2.2.10",
-      "@vaadin/vaadin-grid": "5.5.1",
+      "@vaadin/vaadin-grid": "5.6.0-alpha3",
```

Comment:
Is it fine to have the prerelease versons here?

Reply:
This is harmless, the code is for validating the examples in the docs. The issue is perhaps that it is built against a snapshot version of platform (in this case `16.0-SNAPSHOT`), so recompiling it may cause the frontend dep files to update as the snapshot evolves. I've nevertheless reverted this since it is irrelevant and clutters the commit.